docs/connect-blockchain/other/bitcoin.mdx

```diff
@@ -0,0 +1,220 @@
+  remains cryptographically secure but incorporates additional data or functionality.
+
+- **`address`**: This is the Bitcoin Taproot address derived from the provided public key. It is
+  used to receive Bitcoin.
+
+- **`output`**: This is the output script associated with the address. In Bitcoin transactions, an
+  output script (also known as a scriptPubKey) defines the conditions that must be met to spend the
+  funds sent to an address. For Taproot addresses, the output script enforces the spending
+  conditions defined by the Taproot upgrade.
+
+## Get Balance
+
+```tsx
+/*
+  Use `address` from above
+*/
+const response = await axios.get(`https://blockstream.info/testnet/api/address/${address}/utxo`);
+const utxos = response.data.filter(
+  (utxo: { status: { confirmed: boolean } }) => utxo.status.confirmed,
+);
+const balance = utxos.reduce((acc: any, utxo: { value: any }) => acc + utxo.value, 0);
+console.log("Bitcoin Balance: ", balance);
+```
+
+The code above fetches the UTXOs for the given `address` and filters out the unconfirmed UTXOs. The
+balance is then calculated by summing the value of the UTXOs.
+
+## Send Transaction
+
+```tsx
+/*
+  Use `address`, `output`, `tweakedChildNode`, `xOnlyPubKey`, `utxos` from above.
+*/
+const utxo = utxos[0];
+const feeResponse = await axios.get("https://blockstream.info/testnet/api/fee-estimates");
+// For mainnet use https://blockstream.info/api/fee-estimates
+const maxFee = Math.max(...(Object.values(feeResponse.data) as number[]));
+const fee = maxFee * 1.2;
+if (amount <= fee) {
+  const errorMsg = `Insufficient funds: ${amount} <= ${fee}`;
+  uiConsole(errorMsg);
+  throw new Error(errorMsg);
+}
+
+const sendAmount = amount - Math.floor(fee);
+
+const psbt = new Psbt({ network })
+  .addInput({
+    hash: utxo.txid,
+    index: utxo.vout,
+    witnessUtxo: { value: utxo.value, script: output! },
+    tapInternalKey: xOnlyPubKey,
+  })
+  .addOutput({
+    value: sendAmount,
+    address: "tb1ph9cxmts2r8z56mfzyhem74pep0kfz2k0pc56uhujzx0c3v2rrgssx8zc5q",
+  });
+
+psbt.signInput(0, tweakedChildNode);
+psbt.finalizeAllInputs();
+const txHex = psbt.extractTransaction().toHex();
+
+try {
+  const response = await axios.post(`https://blockstream.info/testnet/api/tx`, txHex);
+  // For mainnet use https://blockstream.info/api/tx
+  console.log("Transaction sent successfully:", response.data);
+  return response.data;
+} catch (error) {
+  console.error("Error sending transaction:", error);
+  throw error;
+}
+```
+
+The code above sends a transaction to the Bitcoin blockchain. The code fetches the fee estimates
+from the Bitcoin blockchain and calculates the fee for the transaction. The code then creates a PSBT
+object and adds the input and output to the PSBT object. The input is signed using the
+`tweakedChildNode` and the PSBT object is finalized. The transaction is then sent to the Bitcoin
+blockchain.
+
+:::note Example
```

Comment:
I think the example should be highlighted more, let's create a section for it at the end rather than using it as an annotation.